src/scss/components/optional/_page-banner.scss

```diff
@@ -71,7 +71,7 @@ $rays--coloured-spacing--wide: 148px;
    }
 
    .c-pageBanner--withRays--white {
-       // Prevent IE from applying these styles as it can't correctly position the rays
+       // @supports prevent IE from applying these styles as it can't correctly position the rays
```

Comment:
To be pedantic – should say someting like – `Prevents IE from applying these styles as IE11 doesn't support the @supports rule`